playwright.config.ts

```diff
@@ -0,0 +1,109 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+import { devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+  testDir: './tests',
+  /* Maximum time one test can run for. */
+  timeout: 30_000,
+  expect: {
+    /**
+     * Maximum time expect() should wait for the condition to be met.
+     * For example in `await expect(locator).toHaveText();`
+     */
+    timeout: 5_000
```

Comment:
5 seconds is long here. You have to wait for 5 seconds each time your test has a bug and doesn't find an element on a page. 1 second should be more than enough time for any of the pages in our demo app to load.

```suggestion
    timeout: 1_000
```